src/online2/online-nnet2-feature-pipeline.cc

```diff
@@ -93,12 +126,26 @@ OnlineNnet2FeaturePipeline::OnlineNnet2FeaturePipeline(
     pitch_ = new OnlinePitchFeature(info_.pitch_opts);
     pitch_feature_ = new OnlineProcessPitch(info_.pitch_process_opts,
                                             pitch_);
-    feature_plus_optional_pitch_ = new OnlineAppendFeature(base_feature_,
-                                                           pitch_feature_);
+    feature_plus_optional_pitch_ = 
+	    new OnlineAppendFeature(feature_plus_optional_cmvn_,
```

Comment:
feature_plus_optional_cmvn_ here and the in the following "else" block seems never been initialized, which leads a SegFault error at Line 144